script.js

```diff
@@ -0,0 +1,67 @@
+document.getElementById("name").addEventListener
```

Comment:
wrap your codes in the  DOMContentLoaded to ensure the DOM is fully loaded before the script runs.
```
document.addEventListener("DOMContentLoaded", function () {
  // All the code is now inside this function
});
```